etna/experimental/prediction_intervals/naive_variance.py

```diff
@@ -0,0 +1,122 @@
+from typing import Sequence
+
+import numpy as np
+import pandas as pd
+import scipy.stats as scs
+
+from etna.datasets import TSDataset
+from etna.experimental.prediction_intervals import BasePredictionIntervals
+from etna.loggers import tslogger
+from etna.pipeline import BasePipeline
+from etna.pipeline.base import _DummyMetric
+
+
+class NaiveVariancePredictionIntervals(BasePredictionIntervals):
+    """Estimate prediction variance based on historical residuals.
+
+    ``NaiveVariancePredictionIntervals`` provides the possibility to estimate prediction quantiles using the following algorithm:
+
+    1. Compute the residuals matrix :math:`r_{it} = \hat y_{it} - y_{it}` using k-fold backtest, where :math:`i` is fold index.
+
+    2. Estimate variance for each step in the prediction horizon :math:`v_t = \\frac{1}{k} \sum_{i = 1}^k r_{it}^2`.
+
+    3. Use :math:`z` scores and estimated variance to compute corresponding quantiles.
+
+    `Reference implementation <https://www.sktime.net/en/stable/api_reference/auto_generated/sktime.forecasting.naive.NaiveVariance.html>`_.
+    """
+
+    def __init__(self, pipeline: BasePipeline, stride: int = 1):
+        """Initialize instance of ``NaiveVariancePredictionIntervals`` with given parameters.
+
+        Parameters
+        ----------
+        pipeline:
+            Base pipeline or ensemble for prediction intervals estimation.
+        stride:
+            Number of points between folds.
+        """
+        if stride <= 0:
+            raise ValueError("Parameter ``stride`` must be positive!")
```

Comment:
I don't think that in the errors we should embrace parameters into double ``, I thought we used only one any other error.